layer1/chain/src/chain_sc.mligo

```diff
@@ -0,0 +1,26 @@
+(*
+    This smart contract is used only as a placeholder to try the /chain/ library. 
+*)
+
+#include "chain.mligo"
+type chain_storage = chain
+type chain_parameter = 
+    | Receive of batch
+    | Remove  of index
+
+(* ENDPOINTS *)
+let main (action, store : chain_parameter * chain_storage) : operation list * chain_storage = 
+    match action with
+    | Receive b -> 
+        begin 
+        match store_batch (b, store) with
+        | Error _ -> failwith "could not store"
+        | Ok c -> [] , c
+        end
+    | Remove i -> [] , remove_batch (i, store)
```

Comment:
for the remove action, I think should add a check to check who is removing. it shouldn't be anyone.

Reply:
It should be no-one, the only point of the endpoint is testing. Is suggest keeping it as is. I'll add a comment in the file, to avoid reproducing the endpoint in final contract.
There might be a case to be made for having some admin endpoints, with extended rights, but it was not the point here.